src/utils/nodeTemplates.ts

```diff
@@ -138,6 +138,13 @@ export function createEmptyTuple(ast: AST): TupleExpression {
   return node;
 }
 
+export function createTuple(ast: AST, nodes: Expression[]): TupleExpression {
```

Comment:
Since there is this function now, I don't think we need to also have `createEmptyTuple` as the only difference would passing an empty list of nodes